core/src/components/textarea/textarea.scss

```diff
@@ -191,3 +189,7 @@
 
   opacity: 1;
 }
+
+:host([auto-grow]) .native-textarea {
```

Comment:
Does this need to account for `auto-grow="false"`?

Reply:
Good point, we should likely update this style and the existing style to explicitly target `:host([auto-grow="true"])`.